examples/mysql_sakila_ingestion/mysql_sakila_ingestion_yamless.py

```diff
@@ -0,0 +1,148 @@
+import datetime as dt
+
+from debussy_concert.core.config.config_environment import ConfigEnvironment
+from debussy_concert.core.config.config_dag_parameters import ConfigDagParameters
+from debussy_concert.data_ingestion.config.rdbms_data_ingestion import ConfigRdbmsDataIngestion
+from debussy_concert.data_ingestion.config.movement_parameters.rdbms_data_ingestion import \
+    RdbmsDataIngestionMovementParameters
+from debussy_concert.data_ingestion.config.movement_parameters.time_partitioned import BigQueryDataPartitioning
+from debussy_concert.data_ingestion.composition.rdbms_ingestion import RdbmsIngestionComposition
+from debussy_concert.core.service.injection import inject_dependencies
+from debussy_concert.core.service.workflow.airflow import AirflowService
+
+config_environment = ConfigEnvironment(
+    project='modular-aileron-191222',
+    region='us-central1',
+    zone='us-central1-a',
+    reverse_etl_bucket='autodelete_dev_bucket',
+    raw_vault_bucket='dotz-datalake-dev-l2-raw-vault',
+    artifact_bucket='dotz-datalake-dev-artifacts',
+    staging_bucket='dotz-datalake-dev-l0-staging',
+    raw_vault_dataset='raw_vault',
+    raw_dataset='raw',
+    trusted_dataset='trusted',
+    reverse_etl_dataset='reverse_etl',
+    temp_dataset='temp',
+    data_lakehouse_connection_id='google_cloud_debussy'
+)
+
+config_dag_parameters = ConfigDagParameters(
+    dag_id="mysql_sakila_ingestion_yamless",
+    default_args={
+        'owner': 'debussy'
+    },
+    description="mysql yamless ingestion example from sakila sample relational database",
+    start_date=dt.datetime(2006, 2, 15),
+    end_date=dt.datetime(2006, 2, 15),
+    catchup=True,
+    schedule_interval='@daily',
+    max_active_runs=1,
+    tags=[
+        'framework:debussy_concert',
+        'project:example',
+        'tier:5',
+        'source:mysql',
+        'type:ingestion',
+        'load:incremental'
+    ]
+)
+
+config_composition = ConfigRdbmsDataIngestion(
+    name='mysql_sakila_ingestion_yamless',
+    description='mysql ingestion yamless',
+    source_type='mysql',
+    source_name='sakila',
+    secret_manager_uri='projects/modular-aileron-191222/secrets/debussy_mysql_dev',
```

Comment:
maybe use the config env for the project
f'projects/{config_environment.project}/secrets/debussy_mysql_dev'